mac/sound.cpp

```diff
@@ -391,11 +391,11 @@ QString CSound::CheckDeviceCapabilities ( const int iDriverIdx )
         if ( AudioObjectSetPropertyData ( audioInputDevice[iDriverIdx], &stPropertyAddress, 0, NULL, sizeof ( Float64 ), &fSystemSampleRate ) !=
              noErr )
         {
-            return QString ( tr ( "Current system audio input device sample "
-                                  "rate of %1 Hz is not supported. Please open the Audio-MIDI-Setup in "
-                                  "Applications->Utilities and try to set a sample rate of %2 Hz." ) )
-                .arg ( static_cast<int> ( inputSampleRate ) )
-                .arg ( SYSTEM_SAMPLE_RATE_HZ );
+            return QString ( tr ( "The selected input device is incompatible "
+                                  "since it does not support the a sample rate of %1 Hz. Please select another "
+                                  "input device or try to set the sample rate to %1 Hz "
+                                  "manually via Audio-MIDI-Setup (in Applications->Utilities)." ) )
+                                  .arg ( QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) );
```

Comment:
I wouldn't say `does not support the a sample rate`... and then `try to set the sample rate` -- it sounds futile.

How about:
`The sample rate on the current input device is not %1.  Please select another device or set the same rate.`

Reply:
Yes. I think the problem here is that some devices don't support setting the sample rate automatically (--> in fact they are compatible with 48kHz) while others are set to a fixed sample rate which is not supported by Jamulus. The user should know:

1. His device might be incompatible --> he needs to use (=buy) another device
2. His device might be compatible but needs to be set to 48kHz manually